Storage/src/ReadStream.php

```diff
@@ -47,7 +47,7 @@ public function __construct(StreamInterface $stream)
      *
      * @return int The size of the stream.
      */
-    public function getSize()
+    public function getSize(): ?int
```

Comment:
The reason this is considered a breaking change is because if someone has made a subclass of `ReadStream` or `WriteStream`, and they defined the `getSize()` method without a return type, then this change would result in a PHP fatal error :

```
Declaration of SomeCustom\ReadStream::getSize() must be compatible with Google\Cloud\Storage\ReadStream::getSize(): ?int
```

To avoid this being a breaking change, we would want to add `final` to the class, or even an `@internal` label, to signal that it's not meant to be subclassed.

In general I think these changes are benign. I don't think anyone is subclassing these, and if they are, an easy fix would be to just add the return types to the subclassed methods.